config/settings.yml

```diff
@@ -41,6 +42,38 @@ plugins:
       - westeurope
       - westus
       - westus2
+  translator_aws_region:
+    default: 'us-east-1'
+    client: true
+    type: enum
+    choices: # Valid choices retrieved from https://docs.aws.amazon.com/general/latest/gr/rande.html#translate_region
```

Comment:
Maybe to keep things simple, we use a simple text field here instead of an enum with choices. The list of AWS region changes often, and this would need updating every time.